packages/nimble/inst/include/nimble/EigenTypedefs.h

```diff
@@ -85,22 +85,72 @@ nimSmartPtr<EIGEN_SVDCLASS_R>   EIGEN_SVD_R(const Eigen::MatrixBase<Derived> &x,
 }
 
 template<class Derived>
-void EIGEN_EIGEN_INTERNAL(const Eigen::MatrixBase<Derived> &x, bool valuesOnly, EIGEN_EIGENCLASS_R *returnClass) {
-  // nimSmartPtr<EIGEN_EIGENCLASS> returnClass = new EIGEN_EIGENCLASS;
-  returnClass->getValues().initialize(0, 0, x.rows());
+bool EIGEN_CHECKSYMMETRY(const Eigen::MatrixBase<Derived> &x) {
+	for(int i = 0; i<x.rows(); i++){
+		for(int j = i + 1; j<x.rows(); j++){
+			if(x(i,j) != x(j,i)){
+				return(false);
```

Comment:
nit: In C++, return is a keyword rather than a function, so `return false;` is more idiomatic.